src/Uno.Toolkit.UI/Controls/ZoomContentControl/ZoomContentControl.xaml.cs

```diff
@@ -0,0 +1,777 @@
+		PointerWheelChanged += OnPointerWheelChanged;
+	}
+
+	private void OnPointerPressed(object sender, PointerRoutedEventArgs e)
+	{
+		if (!IsAllowedToWork)
+		{
+			return; // Don't handle the event when the control is disabled.
+		}
+
+		var pointerPoint = e.GetCurrentPoint(this);
+		var pointerProperties = pointerPoint.Properties;
+
+		// If the middle button of a mouse is pressed, then we want to handle the event.
+		if (pointerProperties.IsMiddleButtonPressed
+#if IS_WINUI
+			&& pointerPoint.PointerDeviceType == PointerDeviceType.Mouse)
+#else
+			&& pointerPoint.PointerDevice.PointerDeviceType == PointerDeviceType.Mouse)
+#endif
+		{
+			e.Handled = true;
+
+			// Capture the pointer so that we can track its movement even if it leaves the bounds of the control.
+			var pointer = e.Pointer;
+			var captured = CapturePointer(pointer);
+			if (captured)
+			{
+				_capturedPointerId = pointer.PointerId;
+				_referencePosition = pointerPoint.Position;
+				_lastPosition = _referencePosition;
+			}
+		}
+	}
+
+	private void OnPointerReleased(object sender, PointerRoutedEventArgs e)
+	{
+		ReleasePointerCaptures();
+		_capturedPointerId = default;
+	}
+
+	private void OnPointerMoved(object sender, PointerRoutedEventArgs e)
+	{
+		if (!(IsAllowedToWork && _capturedPointerId > 0 && IsPanAllowed))
+		{
+			return;
+		}
+
+		var currentPosition = e.GetCurrentPoint(this).Position;
+		_movimentDirection = (currentPosition.X > _lastPosition.X, currentPosition.Y > _lastPosition.Y);
+		_lastPosition = currentPosition;
+
+		if (CanMoveThisWay())
+		{
+			// If the pointer is captured, then we want to handle the event.
+			e.Handled = true;
+
+			var pointerPoint = e.GetCurrentPoint(this);
+
+			// Adjust the offsets based on the pointer's movement.
+			var position = pointerPoint.Position;
+			var deltaX = position.X - _referencePosition.X;
+			var deltaY = position.Y - _referencePosition.Y;
+
+			TryUpdateOffsets(deltaX, deltaY);
+
+			// Update the starting position for next time.
+			_referencePosition = position;
+		}
+	}
+
+	private void TryUpdateOffsets(double deltaX, double deltaY)
+	{
+		if ((deltaX > 0 && CanScrollLeft()) ||
+			(deltaX < 0 && CanScrollRight()))
+		{
+			HorizontalOffset += deltaX;
+		}
+
+		if ((deltaY > 0 && CanScrollUp()) ||
+			(deltaY < 0 && CanScrollDown()))
+		{
+			VerticalOffset += deltaY;
+		}
+	}
+
+	private void OnPointerWheelChanged(object sender, PointerRoutedEventArgs e)
+	{
+		if (!IsAllowedToWork)
+		{
+			return; // Don't handle the event when the control is disabled.
+		}
+
+		var pointerPoint = e.GetCurrentPoint(this);
+		var pointerProperties = pointerPoint.Properties;
+
+		var changeRatio = GetZoomDelta(pointerProperties);
+
+		//Zoom In/Out
+		if (
+#if IS_WINUI
+			pointerPoint.PointerDeviceType == PointerDeviceType.Mouse &&
+#else
+			pointerPoint.PointerDevice.PointerDeviceType == PointerDeviceType.Mouse &&
+#endif
+			e.KeyModifiers.HasFlag(Windows.System.VirtualKeyModifiers.Control) &&
```

Comment:
```
using static Windows.System.VirtualKeyModifiers;
```
```suggestion
			e.KeyModifiers.HasFlag(Control) &&
```

Reply:
Control is ambiguous and conflicts with Microsoft.Xaml.UI.Controls.Control

I'll keep Windows.System.VirtualKeyModifiers.Control for now